particles/Tutorial/Javascript/Demo/source/Game.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+'use strict';
+
+/* global defineParticle */
+// TODO (heimlich@) change to particle
+defineParticle(({DomParticle}) => { 
+
+  return class extends DomParticle {
```

Comment:
Given this particle does not render DOM, maybe we could use `SimpleParticle`, it is an alias to DomParticle. You can see it in a few demos in our codebase.